torch/_functorch/aot_autograd.py

```diff
@@ -1808,16 +2093,8 @@ def aot_dispatch_autograd(flat_fn, flat_args: List[Any], aot_config: AOTConfig):
     class CompiledFunction(torch.autograd.Function):
         compiled_fw = compiled_fw_func
         compiled_bw = None
-        num_outputs = _num_outputs
-        num_outputs_aliased_to_inputs = _num_outputs_aliased_to_inputs
-        num_outputs_aliased_to_intermediates = _num_outputs_aliased_to_intermediates
-        num_outputs_aliased = _num_outputs_aliased
+        metadata = metadata_
```

Comment:
This refactor could have been done in a prep commit and landed super easily. Folded in here the reviewers have to slog through this along with all of the other changes in the diff.

Reply:
Yes. In retrospect, I totally should have done that - sorry about that!